### PR TITLE
Make delete_pod change more prominent in changelog

### PR DESCRIPTION
**Pull request by dstandish**

### CODE REVIEW

File: airflow/providers/cncf/kubernetes/CHANGELOG.rst [hunk 1]
```diff
@@ -25,17 +25,23 @@ Changelog
...
-    the new structure. Additionally ``PodStatus`` enum has been renamed to ``PodPhase``.
+Parameter is_delete_operator_pod has new default
+````````````````````````````````````````````````
+
+Previously, the default for param ``is_delete_operator_pod`` was ``False``, which means that
+after a task runs, its pod is not deleted by the operator and remains on the
+cluster indefinitely.  With this release, we change the default to ``True``.
 
 Notes on changes KubernetesPodOperator and PodLauncher
 ``````````````````````````````````````````````````````
```
Comment (jedcunningham) [inline airflow/providers/cncf/kubernetes/CHANGELOG.rst:41]: I can't suggest it, but I think we want a newline here?

**Review by jedcunningham — APPROVED** [review]

File: airflow/providers/cncf/kubernetes/CHANGELOG.rst (hunk 1 above)
Comment (dstandish) [reply airflow/providers/cncf/kubernetes/CHANGELOG.rst:41]: you mean you want 2 newlines bettween `cluster indef` and `notes on changes`?

Comment (dstandish) [reply airflow/providers/cncf/kubernetes/CHANGELOG.rst:41]: oh after (for some reason in conversation view it looked like diff place)

Comment (jedcunningham) [reply airflow/providers/cncf/kubernetes/CHANGELOG.rst:41]: Between the header (``````) and the first line of the note (.. warning). Maybe it isn't required though 🤷‍♂️

Comment (dstandish) [reply airflow/providers/cncf/kubernetes/CHANGELOG.rst:41]: done

Comment (jedcunningham) [reply airflow/providers/cncf/kubernetes/CHANGELOG.rst:41]: Yep, like that 👍

File: airflow/providers/cncf/kubernetes/CHANGELOG.rst [hunk 2]
```diff
@@ -25,18 +25,25 @@ Changelog
...
 
+.. warning:: Many methods in ``KubernetesPodOperator`` and ``PodLauncher`` have been renamed.
+    If you have subclassed ``KubernetesPodOperator`` will need to update your subclass to reflect
```
Comment (SamWheating) [inline airflow/providers/cncf/kubernetes/CHANGELOG.rst]:
```suggestion
    If you have subclassed ``KubernetesPodOperator`` you will need to update your subclass to reflect
```

**Review by kaxil — APPROVED** [review]

**Pull request merged**